xchange-huobi/src/main/java/org/knowm/xchange/huobi/HuobiAdapters.java

```diff
@@ -76,8 +76,7 @@ public static Ticker adaptTicker(HuobiTicker huobiTicker, CurrencyPair currencyP
   }
 
   static ExchangeMetaData adaptToExchangeMetaData(
-          HuobiAssetPair[] assetPairs, HuobiAsset[] assets,
-          ExchangeMetaData staticMetaData, HuobiCurrencyWrapper[] currencyWrapper) {
+          HuobiAssetPair[] assetPairs, HuobiAsset[] assets, ExchangeMetaData staticMetaData, HuobiCurrencyWrapper[] currencyWrapper) {
```

Comment:
is this necessary to change for this review?

Reply:
I can revert it if desired? Commit comes from the previous hobit push from cloned repository.

Reply:
please just update it so there is no difference here, this isn't anything related to your changes os shouldnt be visible

Reply:
Fixed.